app/models/user.rb

```diff
@@ -17,6 +17,7 @@ class User < ApplicationRecord
   validates :email, presence: true, uniqueness: {case_sensitive: false}
   validates :first_name, :last_name, :status_group, presence: true
   validates :password_set, inclusion: [true, false]
+  validates :personal_api_key, allow_blank: true, length: {maximum: 255}
```

Comment:
Since there could be multiple APIs or API keys, I would probably go with a more specific name hinting that this is for OpenAI.

Reply:
Yup, good point. I will adjust that.

I am pushing a working solution very soon ;)

Reply:
> Yup, good point. I will adjust that.

Thanks 🙏 

> I am pushing a working solution very soon ;)

Awesome, that's great news and pretty fast 💪